### ISSUE TITLE
WinXP Builds

### ISSUE THREAD

**Issue by NEO-SPECTRUMAN** [Bionus/imgbrd-grabber#2992]
I build Grabber 7.2.1 and put there scripts from 7.11.0
https://github.com/NEO-SPECTRUMAN/Imgbrd-Grabber-XP/releases
it systematically crashes but it's usable
(last version that works on XP 5.5.2 and it's MOAR unstable)
![2023-06-25_061235](https://github.com/Bionus/imgbrd-grabber/assets/88002487/6e62b2ed-aac9-49a7-b5dd-a551a063a6ed)

**Comment by Bionus** [comment]
Why still using XP 💀

Any specific reason you built based on 7.2.1 and not 7.3.0 or higher?

**Comment by NEO-SPECTRUMAN** [comment]
> Why still using XP 💀

win15 gui totally unusable and annoying and unconfigurable and etc...
fckn UAC and NO root explorer (dragndrop doesn't works for root apps)
unusable explorer
bluuuured DPI scaling thats anoying 
(and there is no neither neithboor scaling)
on win10 im used 3 gui tweakers
and it's still works like shit..
etc...


> Any specific reason you built based on 7.2.1 and not 7.3.0 or higher?

now i have only Qt 5.5.1
last Qt for XP is 5.6.3 (but it request custom qt libs build)
new versions also looks like with broked internet module or so
last XP Grabber works on Qt 5.7.0
(but it everytime crashes)


7.5.1 moved to Qt 5.14
i patched somethings
but there is still new Qt code that i can fix by now

7.3.2 has some errors
and i just skip it for now

also i builded
5.4.0 5.5.2 6.0.0 7.0.0

**Comment by crlink271986** [comment]
I agree with bionus why use XP it's a "Hackers Playground" plus IE won't work on the modern web KMelon is the best you can get for web browsing anyways

**Comment by teward** [comment]
Windows XP is a long dead OS that has zero support from microsoft, is laden with security holes and hackable vulnerabilities, and it is unfeasible to support something that is well over a decade old and dead upstream.  I don't think Bionus (or this project) should support a now-dead-upstream OS.  My 2 cents.

**Comment by Bionus** [comment]
I mean I agree, and even Windows 7/8 support will be dropped from Grabber soon.

But still, thanks @NEO-SPECTRUMAN for providing a build for those needing it, I know some countries are quite reluctant from moving on from XP 👍

**Comment by NEO-SPECTRUMAN** [comment]
Grabber 7.3.2
https://github.com/NEO-SPECTRUMAN/Imgbrd-Grabber-XP/releases/tag/r0002
it still crashing

**Comment by teward** [comment]
> so WHAT?
> you has zero support from microsoft
> so why you don't go kill yourself? :)

Fun fact: glorifying suicide by recommending it is against Github Terms of Service.  I don't think @Bionus is fond of that type of behavior either.

Enjoy whatever Microsoft doles out on you when they see the report on your post buddy.
